.travis.yml

```diff
@@ -1,20 +1,15 @@
 dist: xenial
 sudo: required
 language: scala
-before_install:
-# TZ needed for Oracle driver!
-- export TZ=Asia/Kamchatka
-- docker version
-- sh -v travis/runcontainer.sh oracle db2
-- docker ps
-- sh -v ./travis/extractNonPublicDeps
-jdk:
-  - openjdk8
-  - openjdk11
-  - openjdk12
-notifications:
-  flowdock:
```

Comment:
Has this flowdock stuff been removed on purpose?

Reply:
Yes. We no longer use Flowdoc.